nilearn/maskers/_mixin.py

```diff
@@ -67,6 +77,80 @@ def set_output(self, *, transform=None):
 
 
 class _LabelMaskerMixin:
+    lut_: pd.DataFrame
+    _lut_: pd.DataFrame
```

Comment:
Do we really have to have these two?

If that is the case, I would suggest to create a private function to get the `lut` where necessary. Something` like` :

```python

def _get_lut(self):
    if hasattr(self, "_lut_"):
        return self._lut_
    else:
        return self.lut_
```